components/ide/jetbrains/image/status/main.go

```diff
@@ -358,12 +359,25 @@ func updateVMOptions(alias string, content string) string {
 	gitpodVMOptions := []string{"-Dgtw.disable.exit.dialog=true"}
 	vmoptions := deduplicateVMOption(ideaVMOptionsLines, gitpodVMOptions, filterFunc)
 
-	// user-defined vmoptions
+	// user-defined vmoptions (EnvVar)
 	userVMOptionsVar := os.Getenv(strings.ToUpper(alias) + "_VMOPTIONS")
 	userVMOptions := strings.Fields(userVMOptionsVar)
 	if len(userVMOptions) > 0 {
 		vmoptions = deduplicateVMOption(vmoptions, userVMOptions, filterFunc)
 	}
+
+	// user-defined vmoptions (.gitpod.yml)
+	gitpodConfig, err := parseGitpodConfig(repoRoot)
```

Comment:
Should not we parse once and pass gitpodConfig to updateVMOptions and installPlugins?